chromium_src/components/bookmarks/browser/bookmark_node.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// other_node should remain hidden
+#define BRAVE_IS_VISIBLE    \
+  if (type() == OTHER_NODE) \
```

Comment:
can't we use `ChromeBookmarkClient::IsPermanentNodeVisible`? That's how MOBILE is displayed on android/ios and hidden on desktop

Reply:
meaning subclass `ChromeBookmarkClient` if we haven't already

Reply:
wait, you already have `BraveBookmarkClient::IsPermanentNodeVisible` so why do we need this?

Reply:
the patch was originally `return visible_` to get rid of `!children().empty()`.
Then I pulled upstream unit test to run and expect it to be just invisible without override `TestBookmarkClient`

Reply:
The reason that other node is not always empty listed here
https://github.com/brave/brave-core/pull/3620/files#r335092116

Reply:
removed in 3f388ed